modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/java/com/liferay/frontend/data/set/admin/web/internal/portlet/action/ImportSystemDataSetMVCResourceCommand.java

```diff
@@ -114,31 +142,296 @@ protected void doTransactionalCommand(
+					"r_dataSetToDataSetListSections_l_dataSetId",
+					objectEntry.getObjectEntryId()
+				).build(),
+				new ServiceContext());
+		}
+
+		if (baseListFDSView.isDefault()) {
+			Map<String, Serializable> values = objectEntry.getValues();
+
+			values.put("defaultVisualizationMode", "list");
+
+			_objectEntryService.updateObjectEntry(
+				objectEntry.getObjectEntryId(), values, new ServiceContext());
+		}
+	}
+
+	private void _addDataSetTableSectionObjectEntries(
+			BaseTableFDSView baseTableFDSView,
+			HttpServletRequest httpServletRequest, ObjectEntry objectEntry)
+		throws Exception {
+
+		Locale locale = _portal.getLocale(httpServletRequest);
+
+		ResourceBundle resourceBundle = ResourceBundleUtil.getBundle(
+			"content.Language", locale, getClass());
+
+		ObjectDefinition dataSetTableSectionObjectDefinition =
+			_objectDefinitionLocalService.
+				fetchObjectDefinitionByExternalReferenceCode(
+					"L_DATA_SET_TABLE_SECTION",
+					_portal.getCompanyId(httpServletRequest));
+
+		FDSTableSchema fdsTableSchema = baseTableFDSView.getFDSTableSchema(
+			locale);
+
+		Map<String, FDSTableSchemaField> fieldsMap =
+			fdsTableSchema.getFDSTableSchemaFieldsMap();
+
+		for (FDSTableSchemaField fdsTableSchemaField : fieldsMap.values()) {
+			String label = fdsTableSchemaField.getLabel();
+
+			if (fdsTableSchemaField.isLocalizeLabel()) {
+				label = LanguageUtil.get(
+					resourceBundle, fdsTableSchemaField.getLabel());
+			}
+
+			if (Validator.isNull(label)) {
+				label = StringPool.BLANK;
+			}
+
+			HashMapBuilder.HashMapWrapper<String, Serializable> values =
+				HashMapBuilder.<String, Serializable>put(
+					"externalReferenceCode",
+					StringBundler.concat(
+						objectEntry.getExternalReferenceCode(), "_",
+						fdsTableSchemaField.getFieldName())
+				).put(
+					"fieldName",
+					StringUtil.removeLast(
+						fdsTableSchemaField.getFieldName(), ".LANG")
+				).put(
+					"label_i18n",
+					HashMapBuilder.put(
+						dataSetTableSectionObjectDefinition.
+							getDefaultLanguageId(),
+						label
+					).put(
+						locale.toLanguageTag(), label
+					).build()
+				).put(
+					"r_dataSetToDataSetTableSections_l_dataSetId",
+					objectEntry.getObjectEntryId()
+				).put(
+					"renderer",
+					() -> {
+						if (fdsTableSchemaField.
+								isContentRendererClientExtension()) {
+
+							/* it is not possible to get client extension ERC from module URL
+							 univocally, so this CX will have to be configured from DSM */
+
+							return null;
+						}
+
+						String contentRenderer =
+							fdsTableSchemaField.getContentRenderer();
+
+						if (Validator.isNotNull(contentRenderer)) {
+
+							// for actionLinkRenderer there is no actionId
+
+							return contentRenderer;
+						}
+
+						return "default";
+					}
+				).put(
+					"rendererType",
+					() -> {
+						if (fdsTableSchemaField.
+								isContentRendererClientExtension()) {
+
+							return "clientExtension";
+						}
+
+						return null;
```

Comment:
I believe the default return here should be `"internal"`, not `null`

Reply:
changed!